wca-regulations.md

```diff
@@ -19,7 +19,7 @@ Original source of the WCA regulations: [www.worldcubeassociation.org/regulation
 WCA Regulations in [PDF format](link:pdf)
 
 ### Source
-Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/cubing/wca-documents).
+Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/thewca/wca-documents).
```

Comment:
While we are at it, could you please fix the url to point to "wca-regulations" instead of the obsolete "wca-document"?

Also would you mind fixing #361 as well? :p